taxi_service/wsgi.py

```diff
@@ -0,0 +1,16 @@
+"""
+WSGI config for taxi_service project.
+
+It exposes the WSGI callable as a module-level variable named ``application``.
+
+For more information on this file, see
+https://docs.djangoproject.com/en/5.0/howto/deployment/wsgi/
+"""
```

Comment:
You should remove such docstrings or comments like `# Create your views here.` etc., no need to keep them